pkg/sdk/client.go

```diff
@@ -18,6 +18,8 @@ type Client struct {
 	db             *sqlx.DB
 	sessionID      string
 	accountLocator string
+	dryRun         bool
```

Comment:
I like adding this dry run here, but it's not ideal. Now we will not get to the logging have the SDK validation errors (but maybe that's also fine). Let's come back to this later, it is for sure fine for now.

Reply:
i agree, i think there could be a better way of doing this. But this is a simple approach. We also don't enable dry run except in this one resource monitor file which isn't even used by the provider, so it is okay.